resources/js/components/attendance/AttendanceKiosk.vue

```diff
@@ -172,29 +172,42 @@
                 }
                 return {
                     title: 'Tap your BuzzCard now',
-                    html: '<p style="font-size: 1.25em">' + teamName + '</p>', // displays team name
+                    html: `<div>
+                            Recording attendance for<br/>
+                            <b style='font-size: 2em'>${teamName}</b><br/><br/>
+                            <span onClick="document.getElementById('stick-checkbox').click()">
+                            <em>Multiple people attending the same team?</em>
+                            </span>
+                            <div class="form-check">
+                              <input class="form-check-input stick-checkbox" type="checkbox" id="stick-checkbox">
+                              <label class="form-check-label" for="stick-checkbox">
+                                Stick to this team
+                              </label>
+                            </div>
+                            </div>`,
                     showCancelButton: true,
                     allowOutsideClick: () => !Swal.isLoading(),
                     backdrop: true,
                     showConfirmButton: false,
-                    input: 'checkbox',
-                    inputValue: this.stickToTeam,
-                    inputPlaceholder: 'Stick to this team',
+                    timer: (sticky) ? 600000 : 30000,
+                    timerProgressBar: true,
                     didOpen: () => {
                         // Remove focus from checkbox
                         document.activeElement.blur();
-                        document.getElementById('swal2-checkbox')
-                          .addEventListener("change", checkboxEventListener.bind(this));
+                        let checkbox = document.getElementById('stick-checkbox');
+                        checkbox.checked = this.stickToTeam;
+                        checkbox.addEventListener("change", checkboxEventListener.bind(this));
+
                         // Add animated contactless card symbol
-                        var cardImg = document.createElement('img');
+                        let cardImg = document.createElement('img');
                         cardImg.src = '/img/Universal_Contactless_Card_Symbol.svg';
-                        var cardDiv = document.createElement('div');
+                        let cardDiv = document.createElement('div');
                         cardDiv.className = 'animated-contactless-card';
                         cardDiv.appendChild(cardImg);
                         Swal.getTitle().parentNode.insertBefore(cardDiv, Swal.getTitle());
                     },
                     didDestroy: () => {
-                      let checkbox = document.getElementById('swal2-checkbox')
+                      let checkbox = document.getElementById('stick-checkbox')
```

Comment:
is `stick` a typo?

Reply:
No, that's the correct `id` of the new checkbox. `stick` as in `stickToTeam`.